exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/conv/DummyConvertFrom.java

```diff
@@ -32,6 +34,8 @@
 @FunctionTemplate(name = "convert_from", scope = FunctionScope.SIMPLE, nulls = NullHandling.NULL_IF_NULL)
 public class DummyConvertFrom implements DrillSimpleFunc {
 
+  @Param VarCharHolder in1;
+  @Param VarCharHolder in2;
   @Output VarBinaryHolder out;
```

Comment:
I think it's not right to put VarBinary as output type for convert_from() function.